AppKit/CPStringDrawing.j

```diff
@@ -59,22 +57,12 @@ CPStringSizeCachingEnabled = YES;
     return [self sizeWithFont:aFont inWidth:NULL];
 }
 
-- (void) _initializeStringSizing
++ (void) initialize
```

Comment:
hi @cacaodev i am not sure how i can move this from the CPStringDrawing category the main class implementation (CPString.j). How could i access file-private globals such as _CPStringSizeIsCanvasSizingInvalid_ from within CPString.j?